selfdrive/car/subaru/interface.py

```diff
@@ -41,7 +41,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, expe
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0., 20.], [0., 20.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.0025, 0.1], [0.00025, 0.01]]
 
-    elif candidate == CAR.IMPREZA:
+    elif candidate in (CAR.IMPREZA, CAR.CROSSTREK):
```

Comment:
why was the crosstrek split out? is this necessary?

Reply:
I tested if the exact values for Crosstrek weight and other physical parameters made any significant difference for longitudinal vs using Impreza values. There is no noticeable difference so I will merge them back to one model.